### PR TITLE
{bio}[foss/2016b] RSeQC v2.6.4

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@verdurin 
```
Irresolvable dependencies encountered:
numpy/1.11.0-foss-2016b-Python-2.7.12,
Pysam/0.9.0-foss-2016b-Python-2.7.12
```

**Comment by boegel** [comment]
@verdurin mismatch on `numpy` version: 1.11.0 vs provided easyconfig with 1.11.1

Why are you including `numpy` separately though, numpy 1.11.1  is already part of the Python 2.7.12 installation?

File: easybuild/easyconfigs/r/RSeQC/RSeQC-2.6.4-foss-2016b-Python-2.7.12-R-3.3.1.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'PythonPackage'
+
+name = 'RSeQC'
+version = '2.6.4'
+versionsuffix = '-Python-%(pyver)s-R-%(rver)s'
+basepyversionsuffix = '-Python-%(pyver)s'
```
Comment (boegel) [inline easybuild/easyconfigs/r/RSeQC/RSeQC-2.6.4-foss-2016b-Python-2.7.12-R-3.3.1.eb]: @verdurin reuse `basepyversionsuffix`?

```python
versionsuffix = basepyversionsuffix + '-R-%(rver)s'
```

File: easybuild/easyconfigs/r/RSeQC/RSeQC-2.6.4-foss-2016b-Python-2.7.12-R-3.3.1.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ uniformity, strand specificity, transcript level RNA integrity etc."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [SOURCEFORGE_SOURCE]
+
+dependencies = [
+    ('R', '3.3.1'),
+    ('numpy', '1.11.0', basepyversionsuffix),
```
Comment (boegel) [inline easybuild/easyconfigs/r/RSeQC/RSeQC-2.6.4-foss-2016b-Python-2.7.12-R-3.3.1.eb]: @verdurin already included in the Python 2.7.12 installation?

**Comment by verdurin** [comment]
I think the `numpy` was inherited from the previous version, where it was required.

I've also updated `Pysam` to a newer release.

**Comment by boegel** [comment]
@verdurin wouldn't it be better to remove the Pysam & numpy easyconfigs included in here, they serve no more purpose?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2540.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/902f6aa940becb76c8fc8c3cf870d6de for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2003.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5787902df56976b04d06a625bd91d4c8 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**